src/core/endpoints/access_manager/grant_token.js

```diff
@@ -1,18 +1,22 @@
-/*       */
-/* eslint camelcase: 0 */
-
-import { GrantTokenInput, GrantTokenObject, ModulesInject } from '../../flow_interfaces';
 import operationConstants from '../../constants/operations';
 
 export function getOperation() {
   return operationConstants.PNAccessManagerGrantToken;
 }
 
+function hasVspTerms(incomingParams) {
```

Comment:
I see `VSP` word in real life!!! :)

Reply:
I wasn't sure if I should use this to be honest, but couldn't figure out anything else :P